src/main/kotlin/gg/essential/elementa/components/ScrollComponent.kt

```diff
@@ -95,31 +143,45 @@ class ScrollComponent @JvmOverloads constructor(
     val verticalOverhang: Float
         get() = max(0f, calculateActualHeight() - getHeight())
 
+
+
+    val mouseScrollLambda: UIComponent.(UIScrollEvent) -> Unit = {
+        if (Window.of(this).version >= ElementaVersion.v5) {
+            // new behavior
+            val scrollDirection = if (!UKeyboard.isShiftKeyDown()) primaryScrollDirection else secondaryScrollDirection
+            scrollDirection?.let { direction ->
```

Comment:
If it's already in a local variable, may as well just use a simple `if (scrollDirection != null) {` instead of introducing another name for the same thing.